addons/website_sale/views/product_tile_templates.xml

```diff
@@ -70,7 +70,7 @@
                 t-value="product_variant or request.env['product.product'].browse(product._get_first_possible_variant_id())"
             />
             <t
-                t-if="variant and variant.image_128"
+                t-if="variant and variant.with_context(bin_size=True).image_variant_1920"
```

Comment:
`image_variant_1920` instead of `image_128` decreases functionality (even if there is no test). I will look into making a more efficient prefecth instead.